packages/mobile/test/values.ts

```diff
@@ -160,4 +160,17 @@ export const mockPaymentRequests: PaymentRequest[] = [
     notified: true,
     type: NotificationTypes.PAYMENT_REQUESTED,
   },
+  {
```

Comment:
Should this second be diff from the one above?

Reply:
not really, I just needed two of them. ID is diff, though.